### PR TITLE
feat(ui-components): add UIPersona

### PR DESCRIPTION
**Pull request by eouin**
- [x] Add UIPersona as a new component

### CODE REVIEW

**Review by tom-sap — DISMISSED** [review]
lgtm

File: packages/ui-components/stories/UIPersona.story.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                size={UIPersonaSize.size72}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size100}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size120}
+            />
```
Comment (devinea) [inline packages/ui-components/stories/UIPersona.story.tsx:43]:
```suggestion
            <UIPersona
                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
                size={UIPersonaSize.size120}
                text={'John Doe'}
                presence={PersonaPresence.online}
                secondaryText={'Software Engineer'}
                tertiaryText={'In a meeting'}
                optionalText={'Available at 4:00pm'}
            />
```
Could be worth showing other options and maybe export from `PersonaPresence` from UI Components

**Review by devinea — COMMENTED** [review]
Thanks @eouin. Looks good. Small comment below.

File: packages/ui-components/stories/UIPersona.story.tsx (hunk 1 above)
Comment (eouin) [reply packages/ui-components/stories/UIPersona.story.tsx:43]: @devinea Done. 
CleanShot 2022-10-27 at 18 15 51

**Review by Klaus-Keller — APPROVED** [review]
@eouin, thanks for making this component available here. 
- code is clear and easy to understand
- test coverage is excellent
- storybook test exists
- I did not build and test this component locally

Approved.

**Review by tom-sap — APPROVED** [review]

**Review by devinea — APPROVED** [review]
Thanks @eouin for the updates. 
You probably want to add a changeset to the updated version is published. 
https://github.com/SAP/open-ux-tools#create-changesets-for-feature-or-bug-fix-branches

**Pull request merged**